docs/security.rst

```diff
@@ -0,0 +1,459 @@
+Report a security issue
+=======================
+
+We take security very seriously.
+Your input and feedback on our security is always appreciated.
+
+You can send urgent or sensitive reports directly to security@twistedmatrix.com
+
+You may use our public key (below) to keep your message safe.
+Provide us with a secure way to respond.
```

Comment:
In practice we seem to use GHSAs as the communication channel for coordinating fixes. Perhaps we should say that?

```suggestion
We will create a GitHub security advisory to coordinate a fix.
Include your GitHub username so that we add you as a collaborator on the advisory.
See `GHSA-c2jg-hw38-jrqq <https://github.com/twisted/twisted/security/advisories/GHSA-c2jg-hw38-jrqq>`_ for an example.
```